README.md

```diff
@@ -418,3 +418,59 @@ Example with directive to bypass proxy for localhost and host.mydomain.com
 ```
 java -Dhttp.proxyHost=webcache.mydomain.com -Dhttp.proxyPort=8080 -Dhttp.noProxyHosts=”localhost|host.mydomain.com”
 ```
+
+## Executing SignalFlow computations
+
+SignalFlow is SignalFx's real-time analytics computation language. The
+SignalFlow API allows SignalFx users to execute real-time streaming analytics
+computations on the SignalFx platform. For more information, head over to our
+Developers documentation:
+
+* `SignalFlow Overview`_
+* `Getting started with the SignalFlow API`_
+
+.. _SignalFlow Overview: https://developers.signalfx.com/docs/signalflow-overview
+.. _Getting started with the SignalFlow API: https://developers.signalfx.com/docs/getting-started-with-the-signalflow-api
+
+Executing a SignalFlow program is very simple with this client library:
+
+```java
+
```

Comment:
rm empty line, and don't need the first-level indent of the entire block of code below. Same applies to other block line 468